bool reassign
task_assign/task[] task_to_ass
task_assign/task[] rech_points
task_assign/robot[] rob_to_ass
task_assign/robot[] rob_in_rech
task_assign/info[] rob_info
task_assign/info[] rech_rob_info
